src/core.py

```diff
@@ -352,6 +352,15 @@ def bayesdb_variable_name(bdb, population_id, generator_id, colno):
     ''', (population_id, generator_id, colno))
     return cursor_value(cursor)
 
+def bayesdb_colno_to_variable_names(bdb, population_id, generator_id):
```

Comment:
Would prefer to avoid adding a new core operation and instead have the user leverage existing core operations:
```
names = dict(zip(
    bayesdb_variable_names(bdb, population_id, generator_id),
    bayesdb_variable_numbers(bdb, population_id, generator_id)))
```
The performance overhead should be negligible.

Reply:
In the past, I found myself looking for the function @jar398 implemented  in core.py. And I have implemented it from scratch multiple times. I think it belongs here and makes the utilities in core.py more complete.

Reply:
Not sure why this is a big deal or what the two of you would like me to do.

Reply:
I needed for this method to be defined somewhere, and putting it in core made more sense to me than putting it in the file where I used it. And Ulli said it had general utility, which makes sense to me.

Reply:
OK.